physionet-django/console/templates/console/user_management.html

```diff
@@ -111,21 +111,21 @@ <h3>Profile</h3>
     </div>
   {% endfor %}
 
-  {% if groups %}
-      <div class="row mb-1">
-        <div class="col-md-3">
-          User Groups:
-        </div>
-        <div class="col-md-9">
-          {% for group in groups %}
-              <a href="{% url 'user_group' group.name %}">{{ group.name }}</a>
-          {% empty %}
-            N/A
-          {% endfor %}
-        </div>
+  <br>
+  <h3>Permission Groups</h3>
+  <hr>
+    <div class="row mb-1">
+      <div class="col-md-6">
+        The user is a member of the following groups:
       </div>
-
-  {% endif %}
+      <div class="col-md-9">
+        {% for group in groups %}
+            <ul><li><a href="{% url 'user_group' group.name %}">{{ group.name }}</a></li></ul>
+        {% empty %}
+            <ul><li>No Groups</li></ul>
```

Comment:
It feels slightly unnecessary to have a "No groups" bullet here. Could we instead say `The user is a member of the following groups:...` if there is one or more groups and something like `The user does not belong to a permission group.` if not?

Reply:
We now have:

```
      <div class="col-md-9">
        <ul>
        {% for group in groups %}
            <li><a href="{% url 'user_group' group.name %}">{{ group.name }}</a></li>
        {% empty %}
            The user does not belong to a permission group.
        {% endfor %}
        </ul>
      </div>
```

For people who aren't in a group, this step will output `The user does not belong to a permission group.` inside `<ul>` tags which I think isn't quite right. I haven't tested, but what about something like this:

```
  <br />
  <h3>Permission groups</h3>
  <hr>
    <div class="row mb-1">
      {% if groups %}
        <div class="col-md-6">
          The user is a member of the following groups:
        </div>
        <div class="col-md-9">
          <ul>
          {% for group in groups %}
              <li><a href="{% url 'user_group' group.name %}">{{ group.name }}</a></li>
          {% endfor %}
          </ul>
        </div>
        {% else %}
        <div class="col-md-6">
          The user does not belong to a permission group.
        </div>
      {% endif %}
    </div>
```

Reply:
I switched over to the new format, it made the else statement make more sense by not including the <ul> tag